app/services/legacy/dataset_import_service.rb

```diff
@@ -11,13 +11,14 @@ def initialize(legacy_dataset, orgs_cache, themes_cache)
   def run
     update_or_create_dataset
     create_inspire_dataset(dataset.id) if dataset.dataset_type == 'inspire'
-    create_resources(dataset)
+    create_datafiles
+    create_documents
   end
 
   def update_or_create_dataset
```

Comment:
@laurentqro (def) update_or_create_dataset is declared twice(?)

Reply:
good catch, fixed